src/systems/systems/src/pneumatic/mod.rs

```diff
@@ -893,8 +903,13 @@ mod tests {
 
     #[test]
     fn pressurised_reservoir_behaves_like_open_valve() {
+        let mut electricity = Electricity::new();
```

Comment:
Its possible to get rid of all of these hacks in tests. Check the tests just added in the loop_updater we just removed those there

Reply:
Oh nice, will add it to my list. I don't think this should be part of this PR.